### DISCUSSION TITLE
Is there any way to create new species?

### DISCUSSION THREAD

**Discussion by Andyvec** [moo-man/WFRP4e-FoundryVTT#1128] [Q&A] [answered]
Hi,
In my campaign there are new custom species for both PJ and NPJs.
I know how to create a new character by hand and fill in it with all the necessary carrear, traits, etc.
Is there any way I can create a new specie with the current modules?

**Comment by moo-man** [comment, marked as answer]
Yes, but it involves writing a script to add values to a configuration object. If you have the core module or RNHD module you can look at them as an example.

**Reply by Andyvec** [reply]
Could you please point out which are the example scipts/files to look for?

**Reply by moo-man** [reply]
`wfrp4e-core/wfrp4e.js` and `wfrp4e-rnhd/scripts/chargen-gnome.js`
